recipe/meta.yaml

```diff
@@ -14,26 +13,22 @@ source:
     - GEOSSTRtreeTest.cpp.patch  # [win]
 
 build:
-  number: 0
+  number: 1
   features:
-    - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py>=35]
+    - vc{{ vc }}  # [win]
+  run_exports:
+    # pretty poor backcompat.  SO name changes each release.
+    #   https://abi-laboratory.pro/tracker/timeline/geos/
+    - {{ pin_subpackage('geos', max_pin='x.x.x') }}
 
 requirements:
   build:
-    - python  # [win]
-    - msinttypes  # [win]
+    - msinttypes  # [win and vc<14]
     - cmake  # [win]
     - automake  # [not win]
     - libtool  # [not win]
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
   run:
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
+    - {{ compiler('c') }}
```

Comment:
Should we list this at build time to in order to avoid using the system `gcc`? Or is that not advisable at the moment, or never?

Reply:
It should be at build time.  It's extremely unusual to have the compiler necessary at runtime.

Reply:
Got it from [here](https://github.com/AnacondaRecipes/geos-feedstock/commit/71baf0ef850002279d62908f047983b45b990a32) :smile: 

I'll move it to `build` as expected.